### PR TITLE
Done a lot of stuff

### PR DESCRIPTION
**Pull request by arturblch**
If you there, whant to show you

### CODE REVIEW

File: src/PlanetWars.py [hunk 1]
```diff
@@ -1,84 +1,85 @@
...
-            
-            if(max == second):
+
+            if (max == second):
                 #p.Owner() stays the same
-                
-                self.logger("{0:4d}: Player {1} defended planet {2}".format(
-                            self._tick, owner, p.ID()))
+
+                self.logger("{0:4d}: Player {1} defended planet {2} (has 0 shiips)".format(
```
Comment (submitteddenied) [inline src/PlanetWars.py:65]: nit: spelling :)

File: src/PlanetWarsProxy.py [hunk 2]
```diff
@@ -151,115 +191,86 @@ def EnemyFleets(self):
...
-             (p.X(), p.Y(), p.Owner(), p.NumShips(), p.GrowthRate())
-        for f in self._fleets:
-            s += "F %d %d %d %d %d %d\n" % \
-             (f.Owner(), f.NumShips(), f.SourcePlanet(), f.DestinationPlanet(), \
-                f.TotalTripLength(), f.TurnsRemaining())
-        return s
-
-    def Distance(self, source, destination):
-        return source.DistanceTo(destination)
+    #  == Save Function ==
```
Comment (submitteddenied) [inline src/PlanetWarsProxy.py:194]: Nit: generally, if you don't need a method, I tend to delete it, it's saved in the git history if you ever need it again :)

File: src/PlanetWarsProxy.py [hunk 3]
```diff
@@ -151,115 +191,86 @@ def EnemyFleets(self):
...
-        
-        for my_planet in pw.MyPlanets(self._playerid):
-            planetsinview.update(my_planet.GetInRange(pw.Planets()))
-            fleetsinview.update(my_planet.GetInRange(pw.Fleets()))
-        
+                planetsinview[planet.ID()] = planet  # View all planets at first step
+
+        for my_entity in pw.MyPlanets(self._playerid) + pw.MyFleets(self._playerid):
+            planetsinview.update(my_entity.GetInRange(pw.Planets()))
+            fleetsinview.update(my_entity.GetInRange(pw.Fleets()))              # View all enttitys if range
```
Comment (submitteddenied) [inline src/PlanetWarsProxy.py:258]: Nit: spelling

**Review by submitteddenied — COMMENTED** [review]
Just a few small comments if you're interested

**Comment by submitteddenied** [comment]
This is so awesome! I'm really glad you got some use out of the project, can you share what you're using it for?

Since this project isn't really being maintained any more, I won't merge the code here, but I'm happy to direct people to your fork from the readme of this project if you like?

**Comment by arturblch** [comment]
Thank you. I done it just for learning.

**Pull request closed without merging**